libafl/src/monitors/mod.rs

```diff
@@ -164,6 +164,16 @@ impl ClientStats {
         (self.executions as f64) / elapsed
     }
 
+    /// Executions per second
+    fn execs_per_sec_pretty(&mut self, cur_time: Duration) -> String {
+        match self.execs_per_sec(cur_time) {
+            value if value >= 1000.0 => { format!("{}", value) }
```

Comment:
Awes!
For human readable, it'd be good to have `k` and `m` execs, too, right?

Reply:
I'm sorry, I do not understand what either k or m refers to.

Reply:
I mean, instead of 14250300 execs/sec we could show 14.25m execs/sec